x/tokenfactory/simulation/genesis.go

```diff
@@ -0,0 +1,54 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+
+	subspacessim "github.com/desmos-labs/desmos/v5/x/subspaces/simulation"
+	subspacestypes "github.com/desmos-labs/desmos/v5/x/subspaces/types"
+	"github.com/desmos-labs/desmos/v5/x/tokenfactory/types"
+)
+
+// RandomizeGenState generates a random GenesisState for x/tokenfactory
+func RandomizeGenState(simState *module.SimulationState) {
+	// Read the subspaces data
+	subspacesGenesisBz := simState.GenState[subspacestypes.ModuleName]
+	var subspacesGenesis subspacestypes.GenesisState
+	simState.Cdc.MustUnmarshalJSON(subspacesGenesisBz, &subspacesGenesis)
+
+	genesis := &types.GenesisState{
+		Params:        types.NewParams(nil),
```

Comment:
It's preferable to make this non-null. This will allow us to test the burning mechanism and ensure there are no errors. However, if we choose this option, we must ensure that we send an adequate number of tokens to the subspace treasury when generating a `MsgCreateDenom` instance. This will prevent any failed tests caused by insufficient funds.

Reply:
Fixed it to 10.